README.md

```diff
@@ -264,6 +265,39 @@ You may choose to either start from the basics of NLP and gradually move to deep
 
 **Note:** Please make sure to implement at least one project once you are fairly comfortable with NLP. You could either use Deep Learning or just stick to the traditional techniques. Or you could do both and compare the two methods :P 
 
+### Machine Learning
+Machine learning is an application of artificial intelligence (AI) that provides systems the ability to automatically learn and improve from experience without being explicitly programmed. Machine learning focuses on the development of computer programs that can access data and use it learn for themselves. In the past decade, machine learning has given us self-driving cars, practical speech recognition, effective web search, and a vastly improved understanding of the human genome.
 
+Machine Learning today is one of the most sought-after skills in the market. A lot of Software Engineers are picking up ML, simply because it is a highly paid skill.
+So, how do you go about Machine Learning?
 
-Topics to come: Machine Learning, Deep Learning, Computer Vision, Cybersecurity and maybe HCI  
+•	First things first - the prerequisites:
+
+o	Basic calculus. In Machine Learning, you’d be working on a lot of optimizations that require knowledge of Calculus. It would be highly recommended that you are aware of functions, limits, differentiation, maxima, minima, etc.
+o	Linear Algebra. When you talk about ML, you will be dealing with matrices and vectors every day. So, knowledge of Linear Algebra is a must. However, you’d also be required to know about other important topics like Eigenvalues and Eigenvectors.
+o	Probability. Most ML algorithms try to “model” the underlying phenomena that generated the observed data. All of this modelling is probabilistic. It is therefore highly recommended that you are comfortable with the theory of Probability.
+
+•	Getting into actual ML:
+
+#### Courses
+
+1.	[Machine Learning]( https://www.coursera.org/learn/machine-learning) by Andrew Ng. on Coursera. It is the best course there is, on this topic, and takes you slowly and efficiently through all the different algorithms like, Regression, Classification, to name a few. Once you finish this about 8-week long course, you should be ready to start taking up small ML projects of your own for practice.
+
+2.	[Machine Learning Crash Course](https://developers.google.com/machine-learning/crash-course/ml-intro) as provided by Google, is also a good way to step into the world of ML. However, it is, as its name suggests, a crash course, and would not go as into depth as the Coursera one does. So, if you’re in a hurry, try this one out, and you’ll be done in almost a week or two!
+
+
```

Comment:
We should also have a course that deals with Python for ML. It could complement the Coursera course

Reply:
Have you personally used the course? Also most Udemy courses are generally available for $10 right or is this course not covered in that discount?

Reply:
No, I haven't used the course yet. The other two I have tried though. There's a 94% off though for 2 days on the Udemy one. Instead of 12,800, we get it for 830. Do you want me to find a cheaper one though?